### ISSUE TITLE
Improve mutation on fluent API methods

### ISSUE THREAD

**Issue by oscarlvp** [STAMP-project/pitest-descartes#97]
Methods to support a fluent API, for example:

```
class A {
 public A m() {
   // do something
  return this;
 }
}
```
are being mutated with `return null`. This leads to a mutation that can be caught easily and does not show an interesting test issue.
In the cases of such methods that could be detected with a static analysis, then:
do not use `return null`
change the method body to `return this`.

**Comment by ipsita0911** [comment]
I have tried to implement a workaround to this enhancement. Please review the pull request https://github.com/STAMP-project/pitest-descartes/pull/126 and let me know how to proceed further.

**Comment by ipsita0911** [comment]
I have updated the pull request. Please have a look on the same. Pull Request -  #126

**Comment by oscarlvp** [comment]
Included in version 1.3
